canonical-web-and-design/charmhub.io
# 55 replace side nav with checkboxes

## Done

- Replace side nav with responsive checkbox filters
- Add `Sort by` on mobile and keep it in sync with the drop-down sorting

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045/store
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/dashboard?sid=5e85aa643c398c44cc03ac9d) - scroll down on zeplin to get to the mobile designs
- Change to mobile layout. Click on `Sort by` link to open/close filter. Select a `sort by` and see the url change. Change to desktop layout and see the same `sot by` is selected in the dropdown
- Change to mobile layout. Click on `Filter` link to open/close filters. Select a `filter` and see the url changing. Click `Reset` button to reset any `filter` selection. Clicking `Show results` will just hide the filter drawer. Change to desktop layout and see the same `filters` are selected on the left hand side


## Issue / Card

Fixes #55 #100

## Screenshots

[if relevant, include a screenshot]
